code/modules/events/gimmick/kudzu.dm

```diff
@@ -105,6 +105,7 @@
 	var/current_stage = 0
 	var/aggressive = 0
 	var/to_spread = 10				//bascially the radius of child kudzu plants that any given kudzu object can create.
+	var/herbicide = FALSE
```

Comment:
having a var and proc with the same name :shelterfrog:

Reply:
Resolved